tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -243,6 +243,49 @@ def test_login_with_general_exception(mocker):
     }
 
 
+@pytest.mark.django_db
+def test_login_with_inactive_user(mocker, api_client, inactive_user):
+    """Login with inactive user should error and return message."""
```

Comment:
This could be something for a different ticket - but this test flow could definitely use some abstraction. I see this is mostly  copied from the tests above but ideally each individual login test should just have to supply a user to an abstracted login function and then assert as needed below.

Reply:
I definitely agree with that. We'll need to put in a cleanup issue